com.unity.render-pipelines.universal/Editor/Lighting/UniversalRenderPipelineLightUI.Skin.cs

```diff
@@ -5,7 +5,7 @@ namespace UnityEditor.Rendering.Universal
 {
     partial class UniversalRenderPipelineLightUI
     {
-        static class Styles
+        private static class Styles
```

Comment:
This seems issue in the docs tooling as well, when no access modifier is specified then private is used. 
It's ok for this PR of couse but this should be followed up. @oleks-k 

[From C# reference](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers): 
> Class and struct members, including nested classes and structs, have private access by default.

Reply:
I don't think the doctools were complaining about this line. I simply added it to make things clearer as some are internal/public and some private.